index.html

```diff
@@ -5132,6 +5132,12 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
     the use of a specific media type.
   </p>
 
+  <p>
+    This media type MUST NOT be used with <dfn>embedded proof</dfn>.
+  </p>
```

Comment:
So I think we should remove this then it looks good to me other than bikeshedding the may/can language in the other statement.

```suggestion
```

Reply:
If we remove this, we should add examples to the spec for the media type, that include `proof`.

Reply:
> If we remove this, we should add examples to the spec for the media type, that include `proof`.

Agreed, and we could mention some of useful framing text that @TallTed provided around subsets, etc.

Reply:
I also, don't expect we will get consensus to do this, since its basically the JSON-LD version of "alg: none".

It will lead to confusion attacks.

Reply:
> I also, don't expect we will get consensus to do this, since its basically the JSON-LD version of "alg: none".

To the contrary, I think telling people that it's safe to use a payload just because it's tagged with `application/credential+ld+json` is much more like `alg: none`.

Reply:
One of the motivations for registering this media type is for use with `cty` property for JWS, see: 

https://transmute-industries.github.io/vc-jws/#example

In these cases, the `proof` is `external`,  and of course `alg none` could still be present in the header.

You proposal leads to this:

```json
{
  "kid": "https://example.edu/issuers/14#key-0",
  "alg": "none", // bad
  "cty": "application/credential+ld+json"
}
```

```json
{
  "@context": [
    "https://www.w3.org/2018/credentials/v1",
    "https://www.w3.org/2018/credentials/examples/v1"
  ],
  "id": "http://example.edu/credentials/3732",
  "type": ["VerifiableCredential", "UniversityDegreeCredential"],
  "issuer": "https://example.edu/issuers/14",
  "issuanceDate": "2010-01-01T19:23:24Z",
  "credentialSubject": {
    "id": "did:example:123",
    "degree": {
      "type": "BachelorDegree",
      "name": "Bachelor of Science and Arts"
    }
  },
  "proof": { "type": "TrustMe" } // bad
}
```

It would be better to not see a `proof` and not see an `alg: none` for the `application/credential+ld+json` media type.

As you can see above, it is extremely confusing.

That example is not a "VerifiableCredential" even though it has type "VerifiableCredential" and both an `embedded` and `external` proof.

Reply:
Hmm, I don't understand... what difference does it make if `proof` shows up there or not for security? What if I send `alg: none` and a credential with no proof field instead? Why would any of these things matter either way?

`alg: none` must always be rejected. Is there a use case where it shouldn't be? If so, what is that use case?

The use case for using `application/credential+ld+json` in the `cty` field in a JWS makes sense to me. But I'd think a spec around that would be written up such that `alg: none` is always rejected. Then, at that point, I don't understand why the presence (or lack of) a proof matters. The protection is over the contents of the JWS.

Reply:
> The use case for using application/credential+ld+json in the cty field in a JWS makes sense to me. But I'd think a spec around that would be written up such that alg: none is always rejected. 

ironically thats exactly what we are trying to do here in the core data model... perhaps `proof` belongs in data integrity instead.

> Then, at that point, I don't understand why the presence (or lack of) a proof matters. The protection is over the contents of the JWS.

Yes, i agree with this... and if we had a work item where I could specify this, I would say `proof` MUST be ignored if its not understood.

Reply:
the presence of `proof` makes `application/verifiable+credential+ld+json` make sense as a _class_ but not as a fact - because the proof itself can be bogus. similar to alg none.

Meaning, anything can claim to be a verifiable credential even if it is not verifiable. The determination of verifiability is done by the verifier.

Reply:
I'm going to resolve this conversation to make review easier.

I don't intend to accept the change suggestion, based on the support observed for:  https://github.com/w3c/vc-data-model/pull/1014/commits/a99fe01b917e5ce160cf78031e2d6e12546d6630